Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ViewBookmarkSystemComponent.h

```diff
@@ -9,13 +9,16 @@
 #pragma once
 
 #include <AzCore/Component/Component.h>
+#include <AzToolsFramework/ActionManager/ActionManagerRegistrationNotificationBus.h>
 #include <Viewport/LocalViewBookmarkLoader.h>
 
 namespace AzToolsFramework
 {
     //! @class ViewBookmarkSystemComponent
     //! @brief System Component that holds functionality for the ViewBookmarks
-    class ViewBookmarkSystemComponent final : public AZ::Component
+    class ViewBookmarkSystemComponent final
+        : public AZ::Component
+        , public AzToolsFramework::ActionManagerRegistrationNotificationBus::Handler
```

Comment:
This seems like it belongs either on GlobalPaintBrushSettingsSystemComponent (since that's at least a part of the paintbrush system), or on ToolsApplication, since it's reflecting a lot of the paintbrush stuff directly. There's no connection that I'm aware of between ViewBookmark and PaintBrush, so this feels pretty out-of-place the way it is.